src/components/Sidebar/Properties/PropertyOwnerHeader.jsx

```diff
@@ -144,51 +237,80 @@ function PropertyOwnerHeader({
 const mapStateToProps = (state, ownProps) => {
```

Comment:
I totally get if you don't have time to do this here but you could use `useSelector` instead of the connect function for more modern syntax

Reply:
I'll look it up, but won't make that update as part of this PR 

We should spend some time together modernizing the UI code in general, maybe sometime this year at least. I feel more confident doing so now that I understand the concept of hooks better :)